dist/22-5490-schema.json

```diff
@@ -25,7 +25,7 @@
         "testReimbursement",
         "licensingReimbursement",
         "tuitionTopUp",
-        "farmCoop"
+        "cooperativeTraining"
```

Comment:
There actually wasn't a front-end change for this, as the front-end is automatically populating the dropdown options from the schema. It sounds like I'll have to make a corresponding front-end change that drops `farmCoop` from the list, and add it back here in the schema for a week, like @lihanli suggested. Tackling that now!